### ISSUE TITLE
Accessibility improvements - PDF

### ISSUE THREAD

**Issue by robinmolen** [open-formulieren/open-forms#5047]
Accessibility improvements for the submission summary PDF.

- [x] 1.1.1 (non-textual content) The logo in the pdf contains a textual alternative, but cannot be accessed. (#5052)
- [ ] 1.3.1 (information and relations) Missing relations between (submission rapport) PDF field labels and values. (#5099)
- [ ] 1.3.1 (information and relations) When a (submission rapport) PDF field doesn't have a values, we should provide a "No information provided" alternative. (This is to ensure that users understand that a field was left empty) (#5099)
- [ ] 2.4.2 (navigation - page title) The title in the PDF meta description contains leading and trailing whitespace (#5099)
